.golangci.yml

```diff
@@ -11,7 +11,7 @@ run:
   tests: true
   build-tags: [""]
   skip-dirs:
-    - pkg/doc/bbs/bbs12381g2pub/internal/kilic/bls12-381
+    - pkg/crypto/bbs12381g2pub/internal/kilic/bls12-381
```

Comment:
would it make sense to have a folder between crypto and bbs12381g2pub

/pkg/crypto/.../bbs12381g2pub

Reply:
ok, I'll add `bbs` in between just like the original path

Reply:
done

Reply:
renamed `bbs` to `primitive`